### PR TITLE
Misc fix

### PR DESCRIPTION
**Pull request by ibadr**
This PR is a set of misc. fixes to make the package up-to-date with Julia 0.5 and with MRAA v1.0.0. I am removing the deps directory and its build script with the assumption that the user will have MRAA v1.0.0 already installed in the system, which can be retrieved using `find_library`.

### CODE REVIEW

File: deps/mraa [hunk 1]
```diff
@@ -1 +0,0 @@
-Subproject commit c65a1de3ba3e5cf19a1a0162420c281d658b9f7c
```
Comment (sjkelly) [inline deps/mraa:1]: Should .gitmodules be removed as well?

File: src/mraa.jl [hunk 2]
```diff
@@ -27,15 +27,15 @@ function set_log_level(p)
...
+    String(ccall((:mraa_get_platform_name, libmraa), Ptr{UInt8}, ()))
 end
 
 function set_priority(p)
     ccall((:mraa_set_priority, libmraa), Cint, (Cuint,), p)
 end
 
 function version()
-    VersionNumber(bytestring(ccall((:mraa_get_version, libmraa), Ptr{Uint8}, ())))
+    VersionNumber(String(ccall((:mraa_get_version, libmraa), Ptr{UInt8}, ())))
```
Comment (sjkelly) [inline src/mraa.jl:38]: If this is done without Compat it may be good to bump REQUIRE to `julia 0.5` as well.

File: deps/mraa (hunk 1 above)
Comment (ibadr) [inline deps/mraa:1]: Yes!

File: src/mraa.jl (hunk 2 above)
Comment (ibadr) [inline src/mraa.jl:38]: It makes sense to bump to 0.5 given the slow JIT performance on ARM in 0.4.

**Comment by sjkelly** [comment]
@ibadr thanks again for your contributions! I've added you as a collaborator here so we can more easily work together.

**Comment by ibadr** [comment]
@sjkelly glad to help! Thanks.

**Pull request merged**